resources/gui/layouts/game/CraftingWindow.json

```diff
@@ -519,6 +519,96 @@
         "Font": "sourcesansproblack,10",
         "TextScale": 1.0
       },
+      "ProductChanceLabel": {
+        "Bounds": "216,82,107,16",
+        "Padding": "0,0,0,0",
+        "AlignmentEdgeDistances": "0,0,0,0",
+        "AlignmentTransform": "0,0",
+        "Margin": "0,0,0,0",
+        "RenderColor": "255,255,255,255",
+        "Alignments": "",
+        "DrawBackground": true,
+        "MinimumSize": "1,1",
+        "MaximumSize": "4096,4096",
+        "Disabled": false,
+        "Hidden": false,
+        "RestrictToParent": false,
+        "MouseInputEnabled": false,
+        "HideToolTip": false,
+        "ToolTipBackground": null,
+        "ToolTipFont": null,
+        "ToolTipTextColor": "",
+        "BackgroundTemplate": null,
+        "TextColor": "255,255,255,255",
+        "HoveredTextColor": "",
+        "ClickedTextColor": "",
+        "DisabledTextColor": "255,90,90,90",
+        "TextAlign": "Left, Top",
+        "TextPadding": "0,0,0,0",
+        "AutoSizeToContents": true,
+        "Font": "sourcesansproblack,10",
+        "TextScale": 1.0
+      },
+      "DestroyMaterialsChanceLabel": {
+        "Bounds": "216,102,159,16",
+        "Padding": "0,0,0,0",
+        "AlignmentEdgeDistances": "0,0,0,0",
+        "AlignmentTransform": "0,0",
+        "Margin": "0,0,0,0",
+        "RenderColor": "255,255,255,255",
+        "Alignments": "",
+        "DrawBackground": true,
+        "MinimumSize": "1,1",
+        "MaximumSize": "4096,4096",
+        "Disabled": false,
+        "Hidden": false,
+        "RestrictToParent": false,
+        "MouseInputEnabled": false,
```

Comment:
Does this need to be true?

Reply:
https://github.com/AscensionGameDev/Intersect-Assets/pull/20/files#r962369232